utils/motorcontroller/SparkMAXFactory.java

```diff
@@ -38,13 +42,33 @@ public static class Config {
 
     }
 
-    public static final Config defaultConfig = new Config();
-    public static final Config defaultFollowerConfig = new Config();
+    public static final Config defaultConfig = new Config(); // For motors where we care about position and velocity
+    public static final Config defaultVelocityConfig = new Config();// For motors were we only care about precise velocity and not precise position 
+    public static final Config defaultPositionConfig = new Config(); // For motors were we only care about precise position and not precise velocity 
+    public static final Config defaultLowUpdateRateConfig = new Config(); // For motors where we neither care about precise velocity or position
+    public static final Config defaultFollowerConfig = new Config(); //For follower motors
 
     static {
         defaultFollowerConfig.STATUS_FRAME_0_RATE_MS = 1000;
         defaultFollowerConfig.STATUS_FRAME_1_RATE_MS = 1000;
         defaultFollowerConfig.STATUS_FRAME_2_RATE_MS = 1000;
+        defaultFollowerConfig.STATUS_FRAME_3_RATE_MS = 300;
+        defaultFollowerConfig.STATUS_FRAME_4_RATE_MS = 120;
+        defaultFollowerConfig.STATUS_FRAME_5_RATE_MS = 1200;
+        defaultFollowerConfig.STATUS_FRAME_6_RATE_MS = 1200;
+
+        defaultVelocityConfig.STATUS_FRAME_1_RATE_MS = 60;
+        defaultPositionConfig.STATUS_FRAME_5_RATE_MS = 600;
+
+        defaultPositionConfig.STATUS_FRAME_2_RATE_MS = 60;
+        defaultPositionConfig.STATUS_FRAME_6_RATE_MS = 600;
+        
+        defaultLowUpdateRateConfig.STATUS_FRAME_1_RATE_MS = 60;
```

Comment:
as we talked, let's leave frames 0 and 1 at the defaults other than for the follower. frame 0 so we have the applied output and frame 1 so we have the velocity and current (https://docs.revrobotics.com/sparkmax/operating-modes/control-interfaces#periodic-status-frames)

And it's worth a comment somewhere explaining why we're doing that